rc.sh

```diff
@@ -124,6 +135,7 @@ startRC () {
   # Download Hazelcast Community jars
   downloadTests
   downloadHazelcast
+  # Download the SQL jar
   downloadSQL
```

Comment:
Just a minor suggestion;
We can also have a server version check here. No need to try downloading sql jar for server version less then `4.1`

Reply:
We will probably use the same "if something is not available, then ignore it" logic in the future for other optional components. It is possible to check the server version, but I think the current way is more robust.